paasta_tools/marathon_serviceinit.py

```diff
@@ -71,6 +71,19 @@ def restart_marathon_job(service, instance, app_id, normal_instance_count, clien
     start_marathon_job(service, instance, app_id, normal_instance_count, client, cluster)
 
 
+def scale_marathon_job(service, instance, app_id, delta, client, cluster):
+    name = PaastaColors.cyan(compose_job_id(service, instance))
+    _log(
+        service=service,
+        line="EmergencyScale: Scaling %s %s to %d instances" % (name, 'down' if delta < 0 else 'up', abs(int(delta))),
```

Comment:
In case the delta the delta is 5 and the current number of instances is 20, on a scale up this will read : 
"Scaling up to 5 instances", while it should read, "Scaling up *by* 5 instances".